README.md

```diff
@@ -1,11 +1,15 @@
-# eslint-plugin-sonarjs [![Build Status](https://travis-ci.org/SonarSource/eslint-plugin-sonarjs.svg?branch=master)](https://travis-ci.org/SonarSource/eslint-plugin-sonarjs)
+# eslint-plugin-sonarjs [![Build Status](https://travis-ci.org/SonarSource/eslint-plugin-sonarjs.svg?branch=master)](https://travis-ci.org/SonarSource/eslint-plugin-sonarjs) [![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=eslint-plugin-sonarjs&metric=alert_status)](https://sonarcloud.io/dashboard?id=eslint-plugin-sonarjs) [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=eslint-plugin-sonarjs&metric=coverage)](https://sonarcloud.io/dashboard?id=eslint-plugin-sonarjs)
 
-SonarJS rules for ESLint
+SonarJS rules for ESLint detecting bugs and suspicious patterns in your code.
+
+_[We also have a plugin for tslint](https://github.com/SonarSource/SonarTS)_
```

Comment:
@stas-vilchik `TSLint`?

Reply:
@stas-vilchik may be better put link to npm to not confuse with SonarTS for SQ?

Reply:
I prefer links to gihub to gain some stars :smirk: